packages/venia-ui/lib/components/MiniCart2/minicart.css

```diff
@@ -0,0 +1,85 @@
+.root {
+    /* The root aside element takes up the whole screen. */
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+
+    /* It is hidden by default. */
+    opacity: 0;
+    visibility: hidden;
+
+    /* It animates to being closed. */
+    transition-duration: 192ms;
+    transition-timing-function: var(--venia-anim-out);
+    transition-property: opacity, visibility;
+
+    /* It sits over all background content. */
+    z-index: 3;
+}
+
+.root_open {
+    composes: root;
+
+    opacity: 1;
+    visibility: visible;
+
+    /* It animates to being open. */
+    transition-duration: 224ms;
+    transition-timing-function: var(--venia-anim-in);
+}
+
+.body {
+    margin-top: 1rem;
+    overflow-y: scroll;
+}
+
+.contents {
+    align-content: space-between;
+    background-color: rgb(var(--venia-background-color));
+    border-radius: 1px;
+    box-shadow: 1px 1px 5px rgb(var(--venia-grey-darker));
+    display: grid;
+    /* The header and footer heights may change based on PWA-116 and PWA-601. */
+    grid-template-rows: 2rem 1fr 8rem;
+    /* These heights may change based on PWA-605 and should be based on 1.5 and 2.5 visible items. */
+    height: 100%;
+    min-height: 31rem;
+    max-height: 45rem;
+    overflow: hidden;
+    padding: 1rem;
+    position: fixed;
+    right: 0;
+    /* This value is equal to the site header height. */
+    top: 3.5rem;
+    /* 360px / 16 root font size = 22.5rem. */
+    width: 22.5rem;
+}
+
+.footer {
+    border-top: solid 0.4rem rgb(var(--venia-grey));
+    overflow: hidden;
+}
+
+.header {
+    border-bottom: solid 0.15rem rgb(var(--venia-grey));
+    overflow: hidden;
+}
+
+.mask {
+    /* The mask takes up the entire screen. */
```

Comment:
I believe the design calls for the mask to obscure the body but not the header. This is a slightly complicated requirement, since shadow interactions start to matter.

Also, keep in mind that clicking the header—or the trigger a second time—must close the dropdown if it's open. The implementation here (clicking on the mask) relies on the mask overlaying the header, so it may need a new approach.

Reply:
Eeesh, yeah good call. This is outstanding pending an alternate approach / design.

Reply:
Resolved with the new design in https://github.com/magento/pwa-studio/pull/2494/commits/f13d4fee0341526df2c7333485e8033f65609e44. There is no mask anymore.